deploy/nginx-debug/deploy.yaml

```diff
@@ -0,0 +1,355 @@
+apiVersion: v1
```

Comment:
I wonder if we don't include this because someone may just see it and try to install it but it will fail since they aren't using the debug image.